gitops/argocd/tektoncd/tekton-config.yaml

```diff
@@ -17,8 +16,14 @@ spec:
     resources:
       # Only prune taskruns until re-reconciling of resources is fixed.
       - taskrun
+    keep: 100
+    # Note: `keep-since` and `keep` options are not available simultaniously to us.
+    # Upcoming versions of Openshift-Pipelines will have this option.
+    # Since tekton-config resources are additive, adding `keep-since` here, would conflict with the `keep`
+    # which comes with default configurations when Openshift-Pipelines is deployed.
+    # To Do: Use `keep-since` option
```

Comment:
Please add the related issue/PR link here for future reference.

Reply:
I was not sure if it's useful, but doesn't harm either. Done